workbench-core/example/infrastructure/integration-tests/support/resources/users/user.ts

```diff
@@ -13,11 +13,11 @@ export default class User extends Resource {
   }
 
   public async activate(): Promise<AxiosResponse> {
-    return this._axiosInstance.post(`${this._api}/activate`);
+    return this._axiosInstance.put(`${this._api}/activate`);
   }
 
   public async deactivate(): Promise<AxiosResponse> {
-    return this._axiosInstance.post(`${this._api}/deactivate`);
+    return this._axiosInstance.put(`${this._api}/deactivate`);
   }
 
   protected async cleanup(): Promise<void> {
```

Comment:
Consider wrapping this in a try/catch and log any errors. We don't want cleanup to bail out on a single resource.

Reply:
done!